Classes_Homework_5/counts.py

```diff
@@ -0,0 +1,95 @@
+import csv
+from re import findall
+from collections import Counter
+
+from exec_utils.configloader import Config
+from modules.file import Files
+
+cnf = Config()
+f = Files()
+
+
+class Counts:
+    def __init__(self, count_in_file, words_csv, letters_csv):
+        self.source = count_in_file  # use this file as source for counts
+        self.words_csv = words_csv
+        self.letters_csv = letters_csv
+
+    def get_text(self):
+        """Open and read file"""
+        path = f.get_path(self.source)
+        with open(path, "r", encoding="utf-8") as file:
+            my_text = file.read()
+        return my_text
+
+    def words(self):
+        """Split text into words"""
+        text = self.get_text()
+        words_list = findall(r'\b[\w\']+\b', text.lower())
+        return words_list
+
+    def words_count(self):
+        """Count words occurrence in list"""
+        words_list = self.words()
+        words_count = Counter(words_list).most_common()
+        return words_count
+
+    def letters(self):
+        """Take string and create list of all letters (lower everything) and list of upper letters"""
+        letters = [i for i in self.get_text() if i.isalpha()]
+        all_lower = list(map(lambda char: char.lower(), letters))
+        upper_letters = list(filter(lambda char: char.isupper(), letters))
+        return all_lower, upper_letters
+
+    def letters_counts(self):
+        """Count letters occurrence in list: return count of any occurrence, upper occurrence and total letters in list."""
+        all_lower, upper_letters = self.letters()
+        count_lower, count_upper = Counter(all_lower), Counter(upper_letters)
+        total = sum(count_lower.values())
+        return count_lower, count_upper, total
+
+    def dicts_by_letter_counts(self):
+        """"Create list of dictionaries, where keys: letter, total count, upper count and % of all letters. Headers configurable."""
+        count_all, count_upper, total = self.letters_counts()
+        letter = cnf.get_values("HEADERS", "letter")
+        all = cnf.get_values("HEADERS", "total")
```

Comment:
`all` is built-in function: https://pyneng.readthedocs.io/ru/latest/book/10_useful_functions/all_any.html

So it'd be better to rename it.